src/main.rs

```diff
@@ -270,7 +270,13 @@ impl Kopium {
                     if scope == "Namespaced" {
                         println!(r#"#[kube(namespaced)]"#);
                     }
-                    if version.subresources.as_ref().is_some_and(|c| c.status.is_some())
+                    if (version.subresources.as_ref().is_some_and(|c| c.status.is_some())
```

Comment:
```suggestion
                    // status should be lnisted as a subresource
                    // but also check for top-level .status for certain non-conforming crds like argo application
                    if (version.subresources.as_ref().is_some_and(|c| c.status.is_some())
```

let's put a comment on why we are doing this

Reply:
Confirmed! :)